### ISSUE TITLE
Suggestion: Add additional sites from FakeNewsWatch

### ISSUE THREAD

**Issue by SeanKilleen** [jacquerie/stop-the-bullshit#3]
http://www.fakenewswatch.com/

**Comment by SeanKilleen** [comment]
Note: some of these are satire and not strictly fake news. If you're drawing a line between them, you'll want to be cautious. (satire vs. fake news has been the topic of some debate, I understand).

**Comment by jacquerie** [comment]
> satire vs. fake news has been the topic of some debate, I understand

Exactly. I should probably write a manifesto to detail the vision I have for this extension, but I can tell you this: I'm not interested in blocking satire websites (e.g. The Onion), even if people occasionally take them seriously. The only thing I want to block are the websites generated by scripts, filled with ads and designed to provoke outrage with the ultimate objective of making their author richer. I don't want to care about political factions, I just want to punish the spammers.
